src/main/kotlin/com/mparticle/kits/RoktKit.kt

```diff
@@ -213,6 +215,7 @@ class RoktKit :
         attributes: Map<String, String>,
     ): Map<String, String> {
         val finalAttributes = mutableMapOf<String, String>()
+        finalAttributes.putAll(attributes)
```

Comment:
The only issue I see with doing it this way is that we won't get kit filtering functionality anymore right? unless I'm mistaken?

Reply:
The code you linked to relates to the placement attribute mapping, as opposed to user attribute filtering, which are 2 different things.  placement attributes might be a customer passing in `first_name` instead of `firstName`, so they want to map that.  User attribute filtering could be the customer doesn't want us to send `gender`, so even if they pass us an attribute via selectPlacements, that could be filtered out before sending using our UI.  That's what you're referring to, right @BrandonStalnaker ?